int32 index
uint64 time_stamp
uint8[10485760] data

uint32 MAX_SIZE = 10485760
